examples/plot_newfitting_comparison_algorithm.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+
+================================================================================
+Fitting benchmark - Performance comparison between different fitting algorithms
+================================================================================
+
+Now with the new fitting module implemented as :py:func:`~radis.tools.new_fitting.fit_spectrum`
+function on September 6 2022, we can experience an easier-to-use, robust and complete spectrum
+fitting.
+
+This example features performance comparison between 17 different fitting methods and algorithms
+of LMFIT, introduced in `LMFIT method list <https://lmfit.github.io/lmfit-py/fitting.html#choosing-different-fitting-methods>`.
+There are 2 variables to be compared: the last residual, and total number of loops required for
+the fitting process. A good fitting method should have small number of loops (robustness) and has
+low residual (fitting accuracy).
+
+"""
```

Comment:
hello @TranHuuNhatHuy . I'm afraid this takes ages to run (remember it will be run on every new commit ) 

Can you move it somewhere else, like in your repo or in https://github.com/radis/radis-benchmark (manual examples) or in https://github.com/radis/radis-examples ?

Reply:
You can mention the example as a link in the docstring of other Gallery examples

Reply:
I have just deleted the file here, and add it to `radis-benchmark` through a [PR](https://github.com/radis/radis-benchmark/pull/13).

When you approve the PR, I will add its link to all other example files.